README.md

```diff
@@ -17,13 +17,31 @@ Install-Package Serilog.Sinks.AzureDataExplorer
 
 ## How to use
```

Comment:
Link to Nuget needs to be removed. We have to use the partner release pipeline to release it